### PR TITLE
Add Localnet Config to Adjust Blocks per Epoch

### PR DESCRIPTION
**Pull request by GheisMohammadi**
## Issue
This PR adds a new config to adjust localnet blocks per epoch.

config file 2.6.4 added for localnet configuration

### CODE REVIEW

File: cmd/config/config.go [hunk 1]
```diff
@@ -102,6 +102,14 @@ func GetDefaultDNSSyncConfig(nt nodeconfig.NetworkType) harmonyconfig.DnsSync {
 	return dnsSync
 }
 
+func GetDefaultLocalnetConfig() harmonyconfig.LocalnetConfig {
+	localnetConfig := harmonyconfig.LocalnetConfig{
+		BlocksPerEpoch:   nodeconfig.GetDefaultLocalnetBlocksPerEpoch(),
+		BlocksPerEpochV2: nodeconfig.GetDefaultLocalnetBlocksPerEpochV2(),
```
Comment (sophoah) [inline cmd/config/config.go:108]: what will be the required changes if one day we decided to no longer have BlocksPerEpochV2 ?

File: cmd/config/flags_test.go [hunk 2]
```diff
@@ -33,7 +33,8 @@ func TestHarmonyFlags(t *testing.T) {
 				"2p/QmRVbTpEYup8dSaURZfF6ByrMTSKa4UyUzJhSjahFzRqNj --ip 8.8.8.8 --port 9000 --network_type=mainn" +
 				"et --dns_zone=t.hmny.io --blacklist=./.hmy/blacklist.txt --min_peers=6 --max_bls_keys_per_node=" +
 				"10 --broadcast_invalid_tx=true --verbosity=3 --is_archival=false --shard_id=-1 --staking=true -" +
-				"-aws-config-source file:config.json --p2p.disc.concurrency 5 --p2p.security.max-conn-per-ip 5",
+				"-aws-config-source file:config.json --p2p.disc.concurrency 5 --p2p.security.max-conn-per-ip 5 -" +
```
Comment (sophoah) [inline cmd/config/flags_test.go:36]: why is there "5 -" ?

File: internal/configs/sharding/localnet.go [hunk 3]
```diff
@@ -11,6 +11,20 @@ import (
...
+		big.NewInt(0), big.NewInt(localnetV1Epoch), params.LocalnetChainConfig.StakingEpoch, params.LocalnetChainConfig.TwoSecondsEpoch,
+	}
+	// Number of shards, how many slots on each , how many slots owned by Harmony
+	localnetV0   Instance
+	localnetV1   Instance
+	localnetV2   Instance
+	localnetV3   Instance
+	localnetV3_1 Instance
+	localnetV3_2 Instance
+	localnetV4   Instance
```
Comment (sophoah) [inline internal/configs/sharding/localnet.go:25]: what happened and what required changes will we have to do if we decide to have a completely different name and network config here ?

File: cmd/config/config.go (hunk 1 above)
Comment (GheisMohammadi) [reply cmd/config/config.go:108]: this will be deprecated then

File: cmd/config/flags_test.go (hunk 2 above)
Comment (GheisMohammadi) [reply cmd/config/flags_test.go:36]: This is related to P2P configuration. It is testing to pass 5 as disc concurrency.

File: cmd/config/config.go (hunk 1 above)
Comment (sophoah) [reply cmd/config/config.go:108]: or even a BlocksPerEpochV3 ?

File: internal/configs/sharding/localnet.go (hunk 3 above)
Comment (GheisMohammadi) [reply internal/configs/sharding/localnet.go:25]: These are the original names and wasn't touched by this PR.

File: cmd/config/config.go (hunk 1 above)
Comment (GheisMohammadi) [reply cmd/config/config.go:108]: This PR helps any new configurations simply can be added later by add a few line of code

**Comment by sophoah** [comment]
@Frozen please review/approve

**Comment by Frozen** [comment]
looks good. Pls show example how to use

**Review by Frozen — APPROVED** [review]

**Comment by mur-me** [comment]
This is beautiful, thanks for moving this to the harmony configuration @GheisMohammadi :pray:

**Pull request merged**